### PR TITLE
Merge font packages into appropriate meta packages

### PR DESCRIPTION
**Pull request by alerque**
I'm not sure about the Amiri related commit in here. This is a two stage problem: first the CTAN / texlive crew have to pickup the latest upstream font, then distros have to package the package. Lumping them together is interesting from a standpoint of seeing how the font is trickling down into distros, but from a packager standpoint it might show an out of date package that is not the maintainer's problem: the CTAN upstream could be at fault.

How have you handled that case before? Should I pull that commit out of this PR?

### CODE REVIEW

File: 800.renames-and-merges/fonts/l.yaml [hunk 1]
```diff
@@ -34,6 +34,13 @@
     - ttf-liberation
     - ttf-liberation-sans-narrow
 
+- setname: "fonts:libertinus"
+  name:
+    - libertinus
+    - libertinus-fonts
+    - otf-libertinus
+    - otf-libertinus-git
```
Comment (AMDmi3) [inline 800.renames-and-merges/fonts/l.yaml]: No need for this one, as `-git` suffix in AUR is handled globally.

File: 800.renames-and-merges/fonts/a.yaml [hunk 2]
```diff
@@ -80,6 +80,7 @@
     - amiri-fonts
     - fonts-hosny-amiri
     - fonts-ttf-amiri
+    - texlive-amiri
```
Comment (AMDmi3) [inline 800.renames-and-merges/fonts/a.yaml]: I don't think this is correct. Texlive packages are not the same as plain fonts packages.

File: 800.renames-and-merges/fonts/m.yaml [hunk 3]
```diff
@@ -77,6 +77,13 @@
     - montecarlo-font
     - montecarlo_fonts
 
+- setname: "fonts:montserrat"
+  name:
+    - julietaula-montserrat-fonts
+    - montserrat
+    - otf-montserrat
+    - ttf-montserrat
```
Comment (AMDmi3) [inline 800.renames-and-merges/fonts/m.yaml]: Looks like it's missing `fonts-otf-julietaula-montserrat` from Sisyphus.

**Comment by alerque** [comment]
Thanks for the tips. I've dropped the commit relating to Amiri's CTAN package and edited the other ones per your notes.

**Pull request merged**